config/routes.rb

```diff
@@ -41,7 +41,7 @@
     resources :contributor_submissions, path: 'contributor-submissions'
     resources :exchange_listings, only: :index, path: 'listings'
     resources :calendar_events, only: :index, path: 'calendar'
-    get '/indicators/:id_or_slug', to: 'indicators#show', as: 'indicators'
+    get '/indicators/:symbol', to: 'indicators#show', as: 'indicators'
```

Comment:
Does order matter? Might be better to put this line after the `/indicators/overview/` entry to ensure that the hardcoded URLs get evaluated first.

Reply:
It's not at all obvious from the way github has presented it, but this one is just `/indicators` whereas the other two are actually `/api/indicators/*`.

Reply:
🤦‍♂ Rookie mistake on my part, thanks for the heads up!